src/modules/autocomplete.js

```diff
@@ -85,7 +85,7 @@ export function autocomplete(options) {
             return response.json();
           }
 
-          throw new Error(response.statusText);
+          throw response;
```

Comment:
Why are we throwing the entire response here?  it's not an error.

Reply:
it is - if not, `response.ok` would be true.